index.html

```diff
@@ -7,177 +7,352 @@
     <link rel="stylesheet" href="styles.css">
   </head>
 
-  <body>
-    <header>
-      <a id="logo">Mark.O</a>
-      <nav>
-        <img src="./logos/Union.jpeg" alt="menue bar" id="menue-button">
+  
+<body>
+  <div class="hero">
```

Comment:
- [Optional] Please resolve the conflict in this branch so you can merge it. :+1: